ui/components/src/ui-components/timeline/TimelineComponent.tsx

```diff
@@ -51,20 +74,24 @@ interface TimelineComponentState {
   currentDuration: number; // current duration in milliseconds
   totalDuration: number;  // total duration in milliseconds
   repeat: boolean; // automatically restart when the timeline is finished playing
+  includeRepeat: boolean; // include the repeat option in the timeline context menu
 }
 
-/** Component used to playback timeline data
+/** [[TimelineComponent]] is used to playback timeline data
  * @public
  */
 export class TimelineComponent extends React.Component<TimelineComponentProps, TimelineComponentState> {
   private _timeLastCycle = 0;
   private _requestFrame = 0;
   private _unmounted = false;
   private _settings: HTMLElement | null = null;
-  private _expandLabel: string = "Expand";
-  private _minimizeLabel: string = "Minimize";
   private _repeatLabel: string = "Repeat";

```

Comment:
Is "Repeat" used? If so, it should be localized too,

Reply:
It is set to the localized value in the constructor. I'll remove the setting here for clarity.